tkn/README.md

```diff
@@ -0,0 +1,19 @@
+# tkn
+
+This task performs operations on Tekton resources using
+[`tkn`](https://github.com/tektoncd/cli).
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/tkn/tkn.yaml
+```
+
+## Inputs
+
+### Parameters
+
+name      | description                                 | default
```

Comment:
Should we do the same for the rest ? (We need templates and a small script like `./create-task` that would create the folder and put some skeleton in there)